docs/website/data/integrations.yaml

```diff
@@ -706,6 +706,46 @@ integrations:
     inventors:
       - sysdig
 
+  pomerium-authz:
+    title: Pomerium Access Proxy
+    description: Pomerium is an identity-aware proxy that enables secure access to internal applications.  OPA implements authorization under the hood.
+    labels:
+      layer: network
+      category: proxy
+    software:
+      - pomerium
+    blogs:
+      - https://www.pomerium.io/posts/2020/04/16/release-0-7/
+    code:
+      - https://github.com/pomerium/pomerium
+
+  coredns-authz:
+    title: CoreDNS Authorization
+    description: CoreDNS is a cloud-native DNS server written in Go.  OPA can be used as a plugin to filter queries and responses.
+    labels:
+      layer: network
+      category: dns
+    software:
+      - coredns
+    code:
+      - https://github.com/coredns/policy
+    inventors:
+      - infoblox
+
+  gluu-gateway-authz:
+    title: Gluu Gateway Authorization
+    description: Gluu Gateway provides API authentication and authorization for websites built on Kong.  Gluu provides an OPA plugin to handle API authorization.
+    labels:
+      layer: network
+      category: gateway
+    software:
+      - gluu
+      - kong
+    code:
+      - https://github.com/GluuFederation/gluu-gateway
+    docs:
```

Comment:
`docs` -> `tutorials`. There is no check for the `docs` title so this won't show up.

Reply:
Removed